### DISCUSSION TITLE
Reconsider Modrinth support?

### DISCUSSION THREAD

**Discussion by bigmanmad** [TechReborn/TechReborn#3185] [Ideas]
Modrinth is more fleshed out now and it would cool for tech reborn to be added.
(https://github.com/TechReborn/TechReborn/issues/2679#issue-1087936589)

**Comment by modmuss50** [comment]
There are no plans to add TR to modrinth.

**Comment by SgtKaneki** [comment]
Just a little push / demand, is there a specific reason why TR won't be on Modrinth?
